### PR TITLE
feat(parent_chain): clarify parent chain text

### PR DESCRIPTION
**Pull request by Milerius**
Task list on both simple and pro view:

- [x] Try a trade BEP20<->SmartChain without BNB activated
- [x] Try a trade BEP20<->SmartChain with 0 BNB balance
- [x] Try a trade SmartChain<->BEP20 without BNB activated
- [x] Try a trade SmartChain<->BEP20 with 0 BNB balance

### CODE REVIEW

**Comment by smk762** [comment]
1)
image

2)
image

3) 
image
Note: BNB was automatically enabled when enabling BEP20 token, had to disable it to get into this state.

4)
image


Some minor grammar tweaks required, will add to line in files view

File: atomic_defi_design/qml/Constants/General.qml [hunk 1]
```diff
@@ -450,13 +450,13 @@ QtObject {
         case TradingError.ReceiveVolumeIsLowerThanTheMinimum:
             return qsTr("%1 volume is lower than minimum trade amount").arg(rel_ticker) + " : " + General.getReversedMinTradeAmount()
         case TradingError.LeftParentChainNotEnabled:
-            return qsTr("%1 parent chain is not enabled").arg(left_ticker)
+            return qsTr("%1 need to be enabled in order to use %2").arg(API.app.portfolio_pg.global_cfg_mdl.get_parent_coin(left_ticker)).arg(left_ticker)
```
Comment (smk762) [inline atomic_defi_design/qml/Constants/General.qml:453]: - `need`
+ `needs`

**Review by smk762 — APPROVED** [review]
Looks good now, thanks.
Potential enhancement - Add "activate {parent_chain}" link / button / tooltip to activate the required parent coin for cases 1 and 3, like we have in simple best orders view. Can make this a separate issue if needed.

**Comment by tonymorony** [comment]
It looks like https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/atomic_defi_design/assets/languages/atomic_defi_en.ts was not updated with these changes

**Pull request merged**